tests/conftest.py

```diff
@@ -15,7 +16,7 @@
 
 def handle_remove_readonly(func, path, exc):  # no cov
```

Comment:
```suggestion
def handle_remove_readonly(func, path, exc_or_info):  # no cov
   exc = exc_or_info[1] if isinstance(exc, tuple) else exc_or_info
```

to simplify the passover